lib/signs_ui_web/router.ex

```diff
@@ -79,6 +83,19 @@ defmodule SignsUiWeb.Router do
   #   pipe_through :api
   # end
 
+  scope "/_flags" do
+    pipe_through([
+      :redirect_prod_http,
+      :accepts_html,
```

Comment:
This isn't necessary (or the `pipeline :accepts_html` above). In this `pipe_through` notice that one of the steps is `:browser`. That means this pipeline will go through the listed pipelines. And if you look higher up in the file at the `pipeline :browser`, notice that `plug(:accepts, ["html"])` is already there.